crates/federate/src/worker.rs

```diff
@@ -46,17 +46,17 @@ static SAVE_STATE_EVERY_TIME: Duration = Duration::from_secs(60);
 /// this delay limits the maximum time until the follow actually results in activities from that community id being sent to that inbox url.
 /// This delay currently needs to not be too small because the DB load is currently fairly high because of the current structure of storing inboxes for every person, not having a separate list of shared_inboxes, and the architecture of having every instance queue be fully separate.
 /// (see https://github.com/LemmyNet/lemmy/issues/3958)
-static FOLLOW_ADDITIONS_RECHECK_DELAY: Lazy<chrono::Duration> = Lazy::new(|| {
+static FOLLOW_ADDITIONS_RECHECK_DELAY: Lazy<chrono::TimeDelta> = Lazy::new(|| {
   if *LEMMY_TEST_FAST_FEDERATION {
-    chrono::Duration::seconds(1)
+    chrono::TimeDelta::try_seconds(1).expect("TimeDelta out of bounds")
```

Comment:
Can't really use errors in here, so I just use expect.

Reply:
Yes expect is fine with hardcoded values.